flask_app/image_analyzer.py

```diff
@@ -25,6 +25,7 @@
 }
 
 top_retailers = ['Amazon', 'Apple', 'Dell', 'Walmart', 'Target', 'Autozone']
+materials = ['plastic', 'cardboard', 'paper', 'carton']
```

Comment:
what is this 'carton'? Do you mind move your const into a const.py or you could just add this constant into the meta.py in PP_API

Reply:
as well as the retailors, checkout the BRANDS in meta.py

Reply:
@kennylee9672 that's a good idea to use brands in meta.py. I created a symbolic link to meta.py in PP_APP, so that if you update meta.py at some point it'll be updated in the flask app as well.

I removed carton as a material and replaced the materials with ones found in meta.py